carmirror/carmirror_test.go

```diff
@@ -355,14 +355,14 @@ func NewMockConnection(maxBatchSize uint) *MockConnection {
 	}
 }
 
-func (conn *MockConnection) GetBlockSender(orchestrator Orchestrator[BatchStatus]) BlockSender[MockBlockId] {
+func (conn *MockConnection) BlockSender(orchestrator Orchestrator[BatchStatus]) BlockSender[MockBlockId] {
```

Comment:
Hmm, I wonder if these should be renamed to something like NewBlockSender/NewStatusSender as they are more than just getters. Is there a go idiom for that?